src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -1,10 +1,9 @@
 import { normalize } from "./helpers";
-import seedrandom from "seedrandom";
+import Seedrandom, { alea as Alea } from "seedrandom";
 import { entropyToMnemonic } from "bip39";
-
 import { Definitions, DeterministicSeedPhrase } from "@ganache/options";
 
-const { alea } = seedrandom;
+const unseededRng = new Seedrandom();
```

Comment:
Per the [documentation](https://www.npmjs.com/package/seedrandom#cjs--nodejs-usage), I believe this should be (it fails with the proper types)

```suggestion
const unseededRng = Seedrandom();
```